SimPPS/RPDigiProducer/test/RPDigiProducerTest_cfg.py

```diff
@@ -0,0 +1,101 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("RPDigiProducerTest")
+
+# Specify the maximum events to simulate
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(10)
+)
+
+# Configure the output module (save the result in a file)
+# Configure the output module
+process.o1 = cms.OutputModule("PoolOutputModule",
+    fileName = cms.untracked.string('file:RPDigiProducerTest_output.root')
+)
+
+
+process.load("SimGeneral.HepPDTESSource.pdt_cfi")
+
+
+# Configure if you want to detail or simple log information.
+# LoggerMax -- detail log info output including: errors.log, warnings.log, infos.log, debugs.log
+# LoggerMin -- simple log info output to the standard output (e.g. screen)
+process.load("Configuration.TotemCommon.LoggerMin_cfi")
+
+
+################## STEP 1
+process.source = cms.Source("EmptySource")
+
+################## STEP 2 - process.generator
+
+# Use random number generator service
+process.load("Configuration.TotemCommon.RandomNumbers_cfi")
+
+# Monte Carlo gun - elastic specific
+energy = "7000"
+import IOMC.Elegent.ElegentSource_cfi
```

Comment:
@mundim @civanch what is this? It does not look a CMSSW package

Reply:
It was a config test designed for totem environment. Changed accordingly, but it can not be run until all PPS simulation is merged because of dependencies on g4SimHit config file that is not yet in the release.